in_toto/models/link.py

```diff
@@ -19,7 +19,6 @@
 """
 
 import attr
-import json
 import securesystemslib.formats
 from in_toto.models.common import Signable
```

Comment:
I see that the "from foo import bar" convention is being used.  Is there a reason why?  This convention is not advised by our guidelines

Reply:
This is not related to this PR. But acknowledged in https://github.com/in-toto/in-toto/issues/145.